tests/gdx-tests/src/com/badlogic/gdx/tests/DecalAlphaTest.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.tests;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.*;
+import com.badlogic.gdx.graphics.g2d.*;
+import com.badlogic.gdx.graphics.g3d.decals.*;
+import com.badlogic.gdx.math.*;
+import com.badlogic.gdx.tests.utils.*;
+import com.badlogic.gdx.utils.viewport.*;
+
+import java.util.*;
+
+public class DecalAlphaTest extends GdxTest {
```

Comment:
This test is a bit complicated, a load test is not really what we need. Maybe a better approach would be to switch from/to different strategy at runtime and see how much FPS we can get (vsync can be disabled manually by testers when needed). Also adding/removing decals manually (via keyboard keys) would be better.

Before performance measurement, what we really need is to test that the strategy works (eg. the stipple effect, etc)